README.md

```diff
@@ -55,100 +38,91 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 My Portfolio <a name="about-project"></a>
+# 📖 **My First Capstone Project** <a name="about-project"></a>
 
-> Hello Microvers, I created it using HTML, CSS
+This is a project that build with the checking tool linter, language HTML/CSS, Bootstrap and JavaScript to have header, footer, feature speakers in dynamic HTML, home and about page, etc.
 
-## 🛠 Built With <a name="built-with"></a>
 
-1-HTML.
-2-CSS.
-3-Liners.
+## 🛠 Built With <a name="built-with">VS Code</a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">HTML</a></li>
-    <li><a href="https://reactjs.org/">CSS</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
+GitHub/Git, VS Code, HTML/CSS, Linter, JavaScript, Bootstrap
 
 ### Key Features <a name="key-features"></a>
 
--Added index.html file.
--Added style.css file.
+- Header & Footer
+- Home Page
+- About Page
+- Feature Speakers with dynamic HTML
+
+
+All of above can be viewed in mobile and desktop responsively.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+### Future Features <a name="future-features"></a>
 
-Comming soon!!!
+- Other pages
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
-## 💻 Getting Started <a name="getting-started"></a>
+### Live Demo <a name="live-demo"></a>
 
-It Will be great for you and your friends!!!
+- https://lerfast.github.io/module1-capstone-project/
 
-To get a local copy up and running, follow these steps.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Prerequisites
+### Video Presentation <a name="video-presentation"></a>
 
-1-Web browser.
-2-Code Editor.
-3.Git-smc.
+- Comming zoom!!!
```

Comment:
- [ ] As requested by the last code reviewer, please 🙏🏽 add the video for professional documentation as requested for the capstone project.

Reply:
![image](https://user-images.githubusercontent.com/111823606/233501536-c1fc3dd6-15cd-44f2-ae43-b434c218c023.png)

Did it!!!